solution/build.sh

```diff
@@ -0,0 +1,157 @@
+#!/bin/bash
+set -e
+set -u
+
+PWD=$(pwd)
+BUILD_ROOT=${PWD}/_build
+ISO_ROOT=${BUILD_ROOT}/root
+IMAGES_ROOT=${ISO_ROOT}/images
+
+PRODUCT_NAME=Zenko
+PRODUCT_LOWERNAME=zenko
+BUILD_TIMESTAMP=$(date -u +"%Y-%m-%dT%H:%M:%SZ")
+BUILD_HOST=$(hostname)
+
+VERSION_SHORT=$(git describe --abbrev=0)
+VERSION_FULL=${VERSION_SHORT}-dev
+GIT_REVISION=$(git describe --long --always --tags --dirty)
+ISO=${BUILD_ROOT}/${PRODUCT_LOWERNAME}-${VERSION_FULL}.iso
+
+DOCKER=docker
+DOCKER_OPTS=
+DOCKER_SOCKET=unix:///var/run/docker.sock
+HARDLINK=hardlink
+OPERATOR_SDK=operator-sdk
+OPERATOR_SDK_OPTS=
+SKOPEO=skopeo
+SKOPEO_OPTS="--override-os linux --insecure-policy"
+OPERATOR_TAG=$(grep /zenko-operator: deps.txt | awk -F ':' '{print $2}')
+
+# grab our dependencies from our deps.txt file as an array
+readarray -t DEP_IMAGES < deps.txt
+
+function clean()
+{
+	rm -rf ${BUILD_ROOT}
+}
+
+function mkdirs()
+{
+	mkdir -p ${ISO_ROOT}
+}
+
+function copy_yamls()
+{
+	sed "s/ZENKO_OPERATOR_TAG/${OPERATOR_TAG}/" config.yaml > ${ISO_ROOT}/config.yaml
+	cp -R -f operator/ ${ISO_ROOT}/operator
+	sed "s/VERSION_FULL/${VERSION_FULL}/" zenko-version-cr.yaml > ${ISO_ROOT}/operator/zenko-version-cr.yaml
+}
+
+function download_dependency_yamls()
+{
+	OPERATOR_DIR=${ISO_ROOT}/operator
+	CRDS_DIR=${OPERATOR_DIR}/crds
+
+	CERT_VERSION=$(grep /cert-manager-controller: deps.txt | awk -F ':' '{print $2}')
+	wget --directory=${CRDS_DIR} https://github.com/jetstack/cert-manager/releases/download/${CERT_VERSION}/cert-manager.yaml
+
+	# combine all zookeeper files into a bundle.yaml
+	ZOOKEEPER_VERSION=$(grep /zookeeper-operator: deps.txt | awk -F ':' '{print $2}')
+	wget -O ${BUILD_ROOT}/zookeeper-operator.tar.gz https://github.com/pravega/zookeeper-operator/archive/v${ZOOKEEPER_VERSION}.tar.gz
+	mkdir ${BUILD_ROOT}/zookeeper-operator
+	tar -C ${BUILD_ROOT}/zookeeper-operator --strip-components=1 -xf ${BUILD_ROOT}/zookeeper-operator.tar.gz
+	find ${BUILD_ROOT}/zookeeper-operator/deploy/ -type f -exec cat {} + >> ${CRDS_DIR}/zookeeper-operator-bundle.yaml
+
+	# combine all kafka files into a bundle.yaml
+	KAFKA_VERSION=$(grep /kafka-operator: deps.txt | awk -F ':' '{print $2}')
+	wget -O ${BUILD_ROOT}/kafka-operator.tar.gz https://github.com/banzaicloud/kafka-operator/archive/${KAFKA_VERSION}.tar.gz
+	mkdir ${BUILD_ROOT}/kafka-operator
+	tar -C ${BUILD_ROOT}/kafka-operator --strip-components=1 -xf ${BUILD_ROOT}/kafka-operator.tar.gz
+	find ${BUILD_ROOT}/kafka-operator/config/base -type f -exec cat {} + >> ${CRDS_DIR}/kafka-operator-bundle.yaml
```

Comment:
I don't believe the `*-bundle.yaml` files will be picked up for deployment (https://github.com/scality/metalk8s/blob/development/2.6/salt/metalk8s/orchestrate/solutions/deploy-components.sls#L12-L38)

Order is somewhat important in the bundle file, so I don't the find/append is always safe (also there should be the separator, `---`, between each manfiest)

Reply:
oh dang ok... should I just build manual yaml files now? Not sure where to place them if I did though. I could create a manual ordering and fix the --- between files. But I have no idea how to resolve the salt stuff, nor the actual ordering required.

Reply:
It seems that I just need to edit the file names to end with `_crd.yaml` for them to be picked up by salt. Not sure if order between bundles matter though, and not sure how to resolve that if it does.

Reply:
Beware that not everything is a CRD.